src/sentry.ts

```diff
@@ -130,11 +129,4 @@ export const setupSentry = ({
     },
     ...deClutterConfig,
   })
-
```

Comment:
there's 1 last reference to spotlight here: https://github.com/codecov/gazebo/blob/007ba8652703d30f1612751397ba6cc6a7873e61/src/sentry.ts#L82 if you have a chance to delete that!